.gitignore

```diff
@@ -8,3 +8,4 @@ node_modules/
 examples/node/svgdump/
 examples/node/pdf2png/*.png
 package-lock.json
+.vscode
```

Comment:
Remove this since it's unrelated to the pull request. We decided not to include editor-specific entries in our `.gitignore` because it would be an endless list. Please add it to your editor's global igore list instead.